.circleci/config.yml

```diff
@@ -47,6 +47,12 @@ jobs:
           name: "Set RUSTFLAGS to fail the build if there are warnings"
           command: echo 'export RUSTFLAGS="-D warnings"' >> $BASH_ENV
       - checkout
+      - run:
+          name: "Install ruby"
```

Comment:
This step can certainly be added to the docker image if necessary

Reply:
Yep, I'd be happy to add it to the docker image, and that would make each individual test run go a little faster.

Reply:
Thanks!

Reply:
Sure thing! (https://github.com/mozilla/uniffi-rs/pull/461)

Reply:
Great; I've pushed the updated docker image, so in theory we can remove this from the CircleCI config and everything should still run and pass 🤞🏻

Reply:
Done 👍 

FYI: I've noticed that docker image is built with rust `1.50.0` but then we install a different version based on `rust-toolchain.toml` (`1.47.0` at the moment). I've been following why this repo uses a specific rust version, but I guess we can cut a bit of time from each test if we build docker image with rust `1.50.0`. I can prepare a PR if this is useful :)

Reply:
> FYI: I've noticed that docker image is built with rust 1.50.0 but then we install a different version based on
> rust-toolchain.toml (1.47.0 at the moment). 

Huh, so we do! I guess the scripting for the docker image there may pre-date our decision to pin to an earlier Rust. If you have a moment to prep a PR, that would be helpful, please do.

Reply:
hey @rfk, https://github.com/mozilla/uniffi-rs/pull/463